src/Breadcrumbs.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import styled from 'styled-components'
+import {COMMON, FLEX, get} from './constants'
+import theme from './theme'
+
+const SELECTED_CLASS = 'selected'
+
+function BreadcrumbBase({className, children, ...rest}) {
+  const classes = classnames(className, 'Breadcrumb')
+  const wrappedChildren = React.Children.map(children, child => <Breadcrumb.Wrapper>{child}</Breadcrumb.Wrapper>)
+  return (
+    <nav className={classes} aria-label="breadcrumb" {...rest}>
+      <ol>{wrappedChildren}</ol>
+    </nav>
+  )
+}
+
+const Breadcrumb = styled(BreadcrumbBase)`
+  display: flex;
+  justify-content: space-between;
+  ${COMMON};
+  ${FLEX};
+  & > ol {
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
```

Comment:
Is there a reason we aren't applying these styles directly to the `ol`? Something like this:

```jsx
<Box as="ol" my={0} pl={0}>
```